src/models/tools/table/table-content.ts

```diff
@@ -36,8 +46,35 @@ export const TableContentModel = types
     changes: types.array(types.string)
   })
   .preProcessSnapshot(snapshot => {
-    if (snapshot && (snapshot as any).columns) {
-      return { isImported: true, changes: convertImportToChanges(snapshot) };
+    const s = snapshot as any;
+    // handle import format
+    if (s && s.columns) {
+      return { isImported: true, changes: convertImportToChanges(s) };
+    }
+    // handle early change formats
```

Comment:
This hasn't been released yet, has it? Do we need to handle early change formats?